debian/changelog

```diff
@@ -1,12 +1,24 @@
-cloud-init (22.4-0ubuntu0~20.04.2) UNRELEASED; urgency=medium
+cloud-init (23.1-0ubuntu0~20.04.1) focal; urgency=medium
 
   * d/patches/retain-netplan-world-readable.patch:
     - Retain original world-readable perms of /etc/netplan/50-cloud-init.yaml.
       Lunar made the config root read-only.
   * refresh patches:
     + debian/patches/expire-on-hashed-users.patch
+  * Upstream snapshot based on 23.1. (LP: #2008230).
+    List of changes from upstream can be found at
+    https://raw.githubusercontent.com/canonical/cloud-init/23.1/ChangeLog
+
+ -- Alberto Contreras <alberto.contreras@canonical.com>  Fri, 24 Feb 2023 12:51:43 +0100
+
+cloud-init (22.4.2-0ubuntu0~20.04.2) focal; urgency=medium
+
+  * d/compat & d/control: revert bump debhelper-comat to v10. Avoid
+    service restarts across package upgrade (LP: #1999159)
+    - d/compat: replaced with compat level 9
+    - d/control: Build-Depends: revert to debhelper >= 9
 
- -- Chad Smith <chad.smith@canonical.com>  Mon, 09 Jan 2023 09:51:18 -0700
+ -- Chad Smith <chad.smith@canonical.com>  Thu, 08 Dec 2022 09:45:11 -0700
 
 cloud-init (22.4-0ubuntu0~20.04.1) focal; urgency=medium
```

Comment:
We are also missing the following entry from upstream/ubuntu/focal-22.4.x:d/changelog
```diff
 
+cloud-init (22.4.2-0ubuntu0~20.04.1) focal; urgency=medium
+
+  * Upstream snapshot based on 22.4.2 upstream release. (LP: #1996645)
+    - List of changes from upstream can be found at
+      https://raw.githubusercontent.com/canonical/cloud-init/22.4.2/ChangeLog
+    - Includes (LP: #1997559, #1844191) not present in 22.4.0.
+
+ -- James Falcon <james.falcon@canonical.com>  Mon, 28 Nov 2022 10:48:49 -0600
+
```

Reply:
I've force pushed this changeset to allow for build and upload